[2025-05-06T10:45:48.519-0300] {local_task_job_runner.py:120} INFO - ::group::Pre task execution logs
[2025-05-06T10:45:48.547-0300] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=non-requeueable deps ti=<TaskInstance: crypto_pipeline.run_etl manual__2025-05-06T13:35:39.145633+00:00 [queued]>
[2025-05-06T10:45:48.552-0300] {taskinstance.py:2073} INFO - Dependencies all met for dep_context=requeueable deps ti=<TaskInstance: crypto_pipeline.run_etl manual__2025-05-06T13:35:39.145633+00:00 [queued]>
[2025-05-06T10:45:48.552-0300] {taskinstance.py:2303} INFO - Starting attempt 2 of 2
[2025-05-06T10:45:48.576-0300] {taskinstance.py:2327} INFO - Executing <Task(BashOperator): run_etl> on 2025-05-06 13:35:39.145633+00:00
[2025-05-06T10:45:48.594-0300] {standard_task_runner.py:63} INFO - Started process 69922 to run task
[2025-05-06T10:45:48.603-0300] {standard_task_runner.py:90} INFO - Running: ['airflow', 'tasks', 'run', 'crypto_pipeline', 'run_etl', 'manual__2025-05-06T13:35:39.145633+00:00', '--job-id', '80', '--raw', '--subdir', 'DAGS_FOLDER/mi_primer_dag.py', '--cfg-path', '/tmp/tmpad1oibrq']
[2025-05-06T10:45:48.606-0300] {standard_task_runner.py:91} INFO - Job 80: Subtask run_etl
[2025-05-06T10:45:48.641-0300] {task_command.py:426} INFO - Running <TaskInstance: crypto_pipeline.run_etl manual__2025-05-06T13:35:39.145633+00:00 [running]> on host facuvega
[2025-05-06T10:45:48.724-0300] {taskinstance.py:2644} INFO - Exporting env vars: AIRFLOW_CTX_DAG_OWNER='airflow' AIRFLOW_CTX_DAG_ID='crypto_pipeline' AIRFLOW_CTX_TASK_ID='run_etl' AIRFLOW_CTX_EXECUTION_DATE='2025-05-06T13:35:39.145633+00:00' AIRFLOW_CTX_TRY_NUMBER='2' AIRFLOW_CTX_DAG_RUN_ID='manual__2025-05-06T13:35:39.145633+00:00'
[2025-05-06T10:45:48.725-0300] {taskinstance.py:430} INFO - ::endgroup::
[2025-05-06T10:45:48.726-0300] {subprocess.py:63} INFO - Tmp dir root location: /tmp
[2025-05-06T10:45:48.727-0300] {subprocess.py:75} INFO - Running command: ['/usr/bin/bash', '-c', 'python /home/facuvega/Descargas/crypto-etl-airflow/scripts/etl.py']
[2025-05-06T10:45:48.745-0300] {subprocess.py:86} INFO - Output:
[2025-05-06T10:45:50.198-0300] {subprocess.py:93} INFO - ⏳ Ejecutando ETL...
[2025-05-06T10:45:50.198-0300] {subprocess.py:93} INFO - ✅ Datos validados correctamente
[2025-05-06T10:45:50.199-0300] {subprocess.py:93} INFO - ETL [DEBUG] BASE_DIR: /home/facuvega/Descargas/crypto-etl-airflow/scripts/db/crypto.db
[2025-05-06T10:45:50.199-0300] {subprocess.py:93} INFO - [INFO] La base de datos ya existe en: /home/facuvega/Descargas/crypto-etl-airflow/scripts/db/crypto.db
[2025-05-06T10:45:50.199-0300] {subprocess.py:93} INFO - ✅ Datos guardados correctamente en la base de datos
[2025-05-06T10:45:50.200-0300] {subprocess.py:93} INFO - ✅ ETL completado correctamente
[2025-05-06T10:45:50.396-0300] {subprocess.py:97} INFO - Command exited with return code 0
[2025-05-06T10:45:50.397-0300] {taskinstance.py:441} INFO - ::group::Post task execution logs
[2025-05-06T10:45:50.416-0300] {taskinstance.py:1205} INFO - Marking task as SUCCESS. dag_id=crypto_pipeline, task_id=run_etl, execution_date=20250506T133539, start_date=20250506T134548, end_date=20250506T134550
[2025-05-06T10:45:50.470-0300] {local_task_job_runner.py:240} INFO - Task exited with return code 0
[2025-05-06T10:45:50.484-0300] {taskinstance.py:3482} INFO - 1 downstream tasks scheduled from follow-on schedule check
[2025-05-06T10:45:50.488-0300] {local_task_job_runner.py:222} INFO - ::endgroup::
